docs/src/prototypes/chatMessages/ControlMessages/GroupControlMessages.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import * as keyboardKey from 'keyboard-key'
+
+import { List, chatBehavior, Accessibility, Flex, Icon } from '@stardust-ui/react'
+import ControlMessage from './ControlMessage'
+
+const overridenChatBehavior: Accessibility<any> = props => {
+  const behaviorData = chatBehavior(props)
+
+  behaviorData.attributes.root = {
```

Comment:
I think using `_.merge()` might be safer than spreading

Reply:
Can't agree that it is safer. Because the 2nd param of `_.merge()` can be any object, there is no type, basically, you can merge the behavior to anything. If I would want to use for that object type as `AccessibilityDefinition` it will require me to fill in required props, for e.g. `FocusZone`'s mode, which would be redundant.